### DISCUSSION TITLE
Media Picker makes the App Crash

### DISCUSSION THREAD

**Discussion by TheMrRight** [you-apps/ConnectYou#393] [General]
### Steps to reproduce

- Open **Connect You**
- Edit a **Contact**
- Add a **Image**
- You will get two option: **Photos** and **Albums**.
![Screenshot_20240210-015200](https://github.com/you-apps/ConnectYou/assets/122231267/99f525e8-a9a1-4951-b26d-1eed3fca8aae)

**Photos**: will give you all the recent shared images on your device. **It doesn't update**.
**Album**: Crashs the App.
![Screenshot_20240210-014211](https://github.com/you-apps/ConnectYou/assets/122231267/21e1baec-27d9-40d6-9279-1b0113befb8d)

### Expected behavior

I should be able to select a image for contact without the app crashing.

### Actual behavior

The App crashes when I try to select a image for contact.

### Connect You version

9.0 (28)

### Android version

11

### Other details

The **Media Picker** app is `com.google.android.providers.media.module`.

I think using the same method _(I think is the right word)_ as used for selecting the Folder for backup.
![Screenshot_20240210-015641](https://github.com/you-apps/ConnectYou/assets/122231267/0b17c374-b9ba-4d93-98cd-5f4041acb9ac)

_I got a funny feeling that it's not the App's fault but mine. I have no clue how I can even do that, but I got a feeling..._

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by TheMrRight** [comment]
Just my luck 🥲

Its the **Media Picker** app. It also crashes on a different app.

Tho, I like the approach of my default **Contacts** app:
![Screenshot_20240210-021430.jpg](https://github.com/you-apps/ConnectYou/assets/122231267/7c16bb97-936a-4caf-8b2a-4615d5f2d7b9)

**Reply by marufmoinuddin** [reply]
did you find out the reason why it happens? I am seeing this anomaly too.

**Comment by Bnyro** [comment]
I'm converting this to a discussion as it appears to be related to your device / OS - I hope someone will be able to help you there.
